rewriter/src/test/java/com/facebook/coresql/rewriter/TestOrderByRewriter.java

```diff
@@ -70,29 +75,34 @@
 
     private void assertOrderByPatternIsMatchedOrUnmatched(String sql, boolean isMatched)
     {
+        AstNode ast = parseStatement(sql);
+        assertNotNull(ast);
+
         Rewriter rewriter = new OrderByRewriter();
+        Optional<RewriteResult> result = rewriter.rewrite(ast);
+
         if (isMatched) {
-            assertTrue(rewriter.rewritePatternIsPresent(sql));
+            assertTrue(result.isPresent());
+            printRewriteResult(sql, result.get().getRewrittenSql());
         }
         else {
-            assertFalse(rewriter.rewritePatternIsPresent(sql));
+            assertFalse(result.isPresent());
         }
     }
 
-    private void rewriteThenPrint(String sql)
+    private void printRewriteResult(String originalSql, String rewrittenSql)
     {
-        RewriteResult result = new OrderByRewriter().rewrite(sql);
-        System.out.println(format("Before --> %s", sql));
-        System.out.println(format("AFTER --> %s", result.getRewrittenSql()));
+        assertNotNull(parseStatement(rewrittenSql));
+        System.out.println(format("Before --> %s", originalSql));
```

Comment:
We don't normally print to std out in tests, nobody would look at them once code is merged. Anything you want to verify should be written in code.

Reply:
Removed.

Done